rpm.mk

```diff
@@ -82,6 +88,14 @@ local-archive: build-cockpit
 	-mkdir -p dist/$(NAME_VERSION)
 	rsync -a --exclude=node_modules --exclude=dist --exclude=__pycache__ --exclude=.git --exclude=rpmbuild . dist/$(NAME_VERSION)
 
+prep_bundle_libdb:
+	/bin/rm -rf dist/sources/libdb_bundle
+	mkdir -p dist/sources/libdb_bundle/SOURCES rpmbuild/SOURCES
+	cd dist/sources/libdb_bundle && curl -LO $(LIBDB_URL)
+	rpmbuild --define "_topdir $(PWD)/dist/sources/libdb_bundle" -rp --nodeps --noprep dist/sources/libdb_bundle/$(LIBDB_SRC_RPM)
```

Comment:
Why can't we build libdb similar to how we do it with jemalloc, i.e. without resorting to rpmbuild?

Reply:
Only because I do not know the URL to upload the tar.bz2 sources  :wink: 
Now if there is such an URL, yes we should better use it!

Reply:
Do you mean just the libdb tarball or libdb+patches?